examples/step-3simplex/doc/results.dox

```diff
@@ -0,0 +1,102 @@
+<h1>Results</h1>
+
+The following figures show the mesh and the result obtained by executing this
+program:
+
+<table align="center" class="doxtable" style="width:65%">
+  <tr>
+    <td>
+        @image html step_3_simplex_0.png
+    </td>
+    <td>
+        @image html step_3_simplex_1.png
+    </td>
+  </tr>
+</table>
+
+Not surprisingly, the result looks as expected.
+
+
+<h3>Possibilities for extensions</h3>
+
+In this tutorial, we presented how to use the deal.II simplex infrastructure
+to solve a simple Poisson problem on a simplex mesh in 2D. In this scope, we
+could only present a small section of the capabilities. In the following, we
+point out further capabilities briefly.
+
+
+<h4>3D meshes and codim-1 meshes in 3D</h4>
+
+An extension to 3D is quite straightforward. Both FE_SimplexP and QGaussSimplex
+are implemented in a dimensional-independent way so that simply replacing
+everywhere dim=2 with dim=3 should work out of the box.
+
+Furthermore, embedding of a 2D mesh consisting of triangles in 3D space is
+possible.
+
+
+<h4>Mixed meshes</h4>
+
+In step-3, we considered meshes only consisting of quadrilaterals. In this
+tutorial, we took a look at the case that the mesh only consists of triangles.
+In the general case (also known as mixed mesh), the mesh consists of both
+cell types. In 3D, meshes might even consist of more cell types, like
+wedges/prisms and pyramids. We consider such meshes in the tutorial step-3mixed.
+
+
+<h4>Alternative finite elements, quadrature rules, and mapping objects</h4>
+
+In this tutorial, we used the most basic finite-element, quadrature-rule, and
+mapping classes. However, more classes are compatible with simplices. The
+following list gives an overview of these classes:
```

Comment:
All of these are used in later tutorial programs. If you make it here from step-3, this is all going to be over your head. I would remove this whole section.